src/index.js

```diff
@@ -185,65 +186,81 @@ function toProto(doc) {
   // params
   coll = doc.alps.descriptor.filter(semantic);
   coll.forEach(function(msg) {
-    rtn += `message ${msg.id}Params {\n`;
+    val = makeSnakeCase(msg.id);
+    rtn += `message ${val}_params {\n`;
     var c = 0;
     c++;
-    rtn += `  string ${msg.id} = ${c};\n`;    
+    rtn += `  string ${val} = ${c};\n`;    
     rtn += '}\n';
   });
   rtn += '\n';
 
   // objects
   coll = doc.alps.descriptor.filter(groups);
   coll.forEach(function(msg) {
-    rtn += `message ${msg.id} {\n`;
+    val = makeSnakeCase(msg.id);
```

Comment:
Pascal case should be applied here as well.

Reply:
OK. will make this change.-- #thanks

Reply:
see above -- i think the snake and pascal casing routines no longer match